lib/command/base.rb

```diff
@@ -312,8 +312,9 @@ def latest_image_from(items, app_name: config.app, name_only: true)
       end
     end
 
-    def latest_image(app = config.app, org = config.org)
+    def latest_image(app = config.app, org = config.org, refresh: false)
       @latest_image ||= {}
+      @latest_image[app] = nil if refresh
```

Comment:
In some places, we don't want `@latest_image[app]` to be cached, so we use `refresh: true` to ensure that the value will be up-to-date.